src/binaryen.cpp

```diff
@@ -25,6 +25,7 @@
 #include <wasm-validator.h>
 
 #include "binaryen.h"
+#include "binaryen-internal.h"
```

Comment:
It's best when headers matching a .cpp file are included as first. Then you avoid situation when the headers misses any include.